### PR TITLE
RM - Created Nested Entry

### PR DESCRIPTION
**Pull request by Diama1**
#674

### CODE REVIEW

**Comment by Diama1** [comment]
wellBeing

File: client/src/assets/scss/_new.scss [hunk 1]
```diff
@@ -4553,8 +4553,24 @@ div.page-activity.ncd {
     }
   }
 }
+div.page-activity.well-being{
```
Comment (anvmn) [inline client/src/assets/scss/_new.scss]: Add space before `{`

File: client/src/assets/scss/_new.scss [hunk 2]
```diff
@@ -4553,8 +4553,24 @@ div.page-activity.ncd {
...
+    height: 30px;
+    left: 5px;
+    line-height: 30px;
+    position: relative;
+    top: -15px;
+    width: 30px;
+  }
+}
+
+div.page-activity.messaging-center{
```
Comment (anvmn) [inline client/src/assets/scss/_new.scss]: Add space before `{`

File: client/src/elm/Translate.elm [hunk 3]
```diff
@@ -19460,6 +19466,11 @@ translateActivePage page =
                     , kinyarwanda = Nothing
                     }
 
+                WellBeingPage ->
+                    { english = "WellBeing"
```
Comment (anvmn) [inline client/src/elm/Translate.elm]: => `Wellbeing`

File: client/src/elm/Pages/PinCode/Model.elm [hunk 4]
```diff
@@ -50,7 +50,7 @@ type MainMenuActivity
     | MenuDashboards
     | MenuCaseManagement
     | MenuDeviceStatus
-    | MenuMessagingCenter
+    | MenuWellBeing
```
Comment (anvmn) [inline client/src/elm/Pages/PinCode/Model.elm]: => `MenuWellbeing`

File: client/src/elm/Pages/Page.elm [hunk 5]
```diff
@@ -196,6 +196,7 @@ type UserPage
     | TraceContactPage AcuteIllnessTraceContactId
     | PatientRecordPage PatientRecordInitiator PersonId
     | MessagingCenterPage
+    | WellBeingPage
```
Comment (anvmn) [inline client/src/elm/Pages/Page.elm]: => `WellbeingPage`

**Review by anvmn — CHANGES_REQUESTED** [review]

File: client/src/elm/Pages/Router.elm [hunk 6]
```diff
@@ -379,6 +382,7 @@ parser =
         , map (\id -> UserPage <| TraceContactPage id) (s "trace-contact" </> parseUuid)
         , map (\id initiator -> UserPage <| PatientRecordPage initiator id) (s "patient-record" </> parseUuid </> parsePatientRecordInitiator)
         , map (UserPage MessagingCenterPage) (s "messaging-center")
+        , map (UserPage WellBeingPage) (s "well-being")
```
Comment (anvmn) [inline client/src/elm/Pages/Router.elm]: => `wellbeing`

File: client/src/elm/Pages/Router.elm [hunk 7]
```diff
@@ -320,6 +320,9 @@ pageToFragment current =
                 MessagingCenterPage ->
                     Just "messaging-center"
 
+                WellBeingPage ->
+                    Just "well-being"
```
Comment (anvmn) [inline client/src/elm/Pages/Router.elm]: => `wellbeing`

File: client/src/elm/Pages/WellBeing/View.elm [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+        resilienceMessageButton =
+            button
+                [ class "ui primary button"
+                , onClick <| SetActivePage <| UserPage MessagingCenterPage
+                ]
+                [ span [ class "text" ] [ text <| translate language Translate.ResilienceMessage ]
+                , span [ class "counter" ] [ text <| String.fromInt numberOfUnreadMessages ]
+                ]
+    in
+    div [ class "page-activity well-being" ]
```
Comment (anvmn) [inline client/src/elm/Pages/WellBeing/View.elm]: => `wellbeing`

File: client/src/assets/scss/_new.scss (hunk 1 above)
Comment (anvmn) [inline client/src/assets/scss/_new.scss]: => `wellbeing`

**Review by anvmn — CHANGES_REQUESTED** [review]

**Review by anvmn — APPROVED** [review]

**Pull request merged**